Makefile

```diff
@@ -109,4 +109,7 @@ expand_website_templates:
 .PHONY: expand_website_templates
 
 update_contributors_list:
-	cd .github/contributors && npm run all
\ No newline at end of file
+	cd .github/contributors && npm run all
+
+trigger-netlify:
```

Comment:
nits: trigger-netlify might imply that running this make target will kick off netlify  deployment.

How about just _update_netlify_state_ or _updaste_netlify_hash?

Reply:
I like `update_netlify_state`. let me change it

Reply:
Done